hal/src/main/native/cpp/jni/HAL.cpp

```diff
@@ -93,7 +96,7 @@ Java_edu_wpi_first_hal_HAL_terminate
   (JNIEnv*, jclass)
 {
 #ifdef __FRC_ROBORIO__
-  std::abort();
+  ::raise(SIGILL);
```

Comment:
It's `SIGILL`, not `SIGKILL`? (the `K` is missing)

Reply:
Whoops, typo.  Thanks for catching this.

Reply:
Happy to help!